### PR TITLE
Remove packaging and remove test logs

### PR DESCRIPTION
**Pull request by robinjoseph08**
The packaging API is deprecated and was causing issues on the build so I just removed it. I also removed a lot of unnecessary logging that occurred during testing, making it harder to parse when reading travis.

@pon @dmlittle

### CODE REVIEW

File: src/main/java/com/lob/client/AsyncLobClient.java [hunk 1]
```diff
@@ -424,7 +419,7 @@ private BoundRequestBuilder post(final String resourceUrl, final HasLobParams ha
             final Class<T> clazz,
             final BoundRequestBuilder request,
             final ExecutorService callbackExecutorService) {
-            final SettableFuture<T> guavaFut = SettableFuture.create();
+        final SettableFuture<T> guavaFut = SettableFuture.create();
```
Comment (dmlittle) [inline src/main/java/com/lob/client/AsyncLobClient.java]: do we need this?

**Comment by dmlittle** [comment]
LGTM

File: src/main/java/com/lob/client/AsyncLobClient.java (hunk 1 above)
Comment (robinjoseph08) [inline src/main/java/com/lob/client/AsyncLobClient.java]: I assume so. I just fixed indentation.

**Comment by eanderle** [comment]
Yeah, the logging is annoying, but we need it in order to get to 100% test coverage, as it's the only way to exercise all the `toString()` methods.

**Comment by pon** [comment]
Is there a more explicit way to test toString() that validates the response?

**Comment by eanderle** [comment]
Not really. `toString()` exists solely for the purposes of logging human-readable output, and it is extremely bad practice to code any functionality around it, so there's really nothing to validate.

We could have a single test that just goes and prints every single object, but that doesn't seem much better than what we have now.

**Pull request merged**